package.json

```diff
@@ -39,6 +39,8 @@
     "prepublish": "",
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "build-css": "npx node-sass --include-path scss scss/meta.scss css/meta.css",
+    "watch-css": "npx nodemon -e scss -x \"npm run build-css\"",
     "dist": "shx mkdir -p scss/ && shx cp -r src/scss/meta.scss scss/",
```

Comment:
Hey @dantrevino, nice work sir..

I just noticed we need to ensure the css directory and minified css file get packaged up along with the scss files so that when we publish to npm we users receive everything in one clean package. 😄

Do you want to take another stab at this?